client/components/deposits-status/index.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * External dependencies
+ */
+import { Icon } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+
+const DepositsEnabled = ( props ) => {
+	const { depositsEnabled, iconSize } = props;
```

Comment:
Since this component is called `DepositsEnabled` (or `DepositsStatus`) - what do you think about renaming `depositsEnabled` to just `isEnabled`, since the "deposits" scope is already inferred by the component's name?